### DISCUSSION TITLE
select unselectable text

### DISCUSSION THREAD

**Discussion by aprogergely** [community/community#114725] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I was playing a game on my phone through an emulator called Lemuroid, and accidentally pressed a button, which highlighted the text in the conversation currently ongoing on the game screen and allowed me to copy-paste it elsewhere, but I dont remember how I did that

**Comment by ethanwater** [comment, marked as answer]
This really isn't relevant. You should contact Lemuroid themselves. I truly don't understand why you posted it here under the profile category?

**Reply by aprogergely** [reply]
I apologize, I was trying to contact Lemuroid, but it seems my comment went to the wrong place.

**Reply by ethanwater** [reply]
Dw about it! Good luck.
